react/features/prejoin/components/web/Prejoin.tsx

```diff
@@ -432,8 +432,9 @@ const Prejoin = ({
                         visible = { showJoinByPhoneButtons }>
                         <ActionButton
                             OptionsIcon = { showJoinByPhoneButtons ? IconArrowUp : IconArrowDown }
-                            ariaDropDownLabel = { t('prejoin.joinWithoutAudio') }
-                            ariaLabel = { t('prejoin.joinMeeting') }
+                            ariaDropDownLabel = { showJoinByPhoneButtons
+                                ? t('prejoin.joinWithoutAudioMenuClose') : t('prejoin.joinWithoutAudioMenuOpen') }
+                            ariaLabel = { t('prejoin.joinWithoutAudio') }
```

Comment:
I don't agree with changing the aria-label for this button. First, this button is for joining a meeting. Joining without audio is something that a user decides after interacting with the dropdown menu.